trl/data_utils.py

```diff
@@ -478,6 +479,67 @@ def pack_examples(examples: dict[str, list[list]], seq_length: int) -> dict[str,
     return examples
 
 
+@numba.njit(["(int32[:], int32)", "(int64[:], int64)"], cache=True)
```

Comment:
do you mind reverting this change? so that we keep PRs separate

Reply:
actually that might not be that easy, your 2 PRs are quite intertwined right?

Reply:
yep, but I will think about reverting, sorry that I haven't done that before
this should be possible